### PR TITLE
Add pull request number to struct for access via custom template

### PR DESCRIPTION
**Pull request by clintberry**

### CODE REVIEW

**Comment by tboerger** [comment]
LGTM

**Comment by clintberry** [comment]
I think because I closed and re-opened the pull request it broke the LGTM check. Should I close this one and make a new one?

**Comment by donny-dont** [comment]
@clintberry LGTM needs to be removed from all these repos as the service is discontinued.

@bradrydzewski will need to do something to resolve it.

**Comment by tboerger** [comment]
Maybe trust the Github approval system :)

File: DOCS.md [hunk 1]
```diff
@@ -146,6 +146,9 @@ build.created
 build.started
 : unix timestamp for build started
 
+build.pull
+: pull request number (empty string if not a pull request)
+
```
Comment (bradrydzewski) [inline DOCS.md]: docs moved to https://github.com/drone/drone-plugin-index/blob/master/content/drone-plugins/drone-slack/index.md

**Comment by fsouza** [comment]
@clintberry are you still interested in getting this working? Is it ok for me to pick-up from where you left and open a new PR? :D

**Comment by tboerger** [comment]
@fsouza please do it.

**Comment by fsouza** [comment]
@tboerger cool, thanks!

**Comment by tboerger** [comment]
Integrated via https://github.com/drone-plugins/drone-slack/pull/66 but thank you for your initial work.

**Pull request closed without merging**